pyvista/core/pointset.py

```diff
@@ -1359,22 +1358,23 @@ def _from_arrays(
         offset : any, default None
             Ignored (this is a pre-VTK9 legacy).
 
-        cells : sequence or numpy.ndarray dtype=np.int64
+        cells : sequence[int]
             Array of cells.  Each cell contains the number of points in the
             cell and the node numbers of the cell.
 
-        cell_type : sequence or numpy.ndarray np.uint8
+        cell_type : sequence[int]
             Cell types of each cell.  Each cell type numbers can be found from
-            vtk documentation.  See example below.
+            vtk documentation.  More efficient if using ``np.uint8``. See
+            example below.
 
-        points : sequence or numpy.ndarray
+        points : sequence[float]
             Numpy array containing point locations.
 
-        deep : bool, optional
+        deep : bool, default: True
             When ``True``, makes a copy of the points array.  Default
             ``False``.  Cells and cell types are always copied.
 
-        force_float : bool, optional
+        force_float : bool, default: False
             Casts the datatype to ``float32`` if points datatype is
             non-float.  Default ``True``. Set this to ``False`` to allow
             non-float types, though this may lead to truncation of
```

Comment:
Actual default is `True`, matching description. Won't suggest and edit here because removing the default from the description will need rewrapping anyway.

Reply:
Resolved in 367d26a111e306d978c835b03f1441d8472fe96d.